datalad/distribution/create_sibling.py

```diff
@@ -163,6 +157,27 @@ def _create_dataset_sibling(
                 lgr.info(_msg + " Skipping")
                 return
             elif existing == 'replace':
+                if path_children:
+                    has_git = '.git' in path_children
+                    _msg_stats = _msg \
```

Comment:
I'd prefer a message that states the actual path it is about to remove rather than calling it "it". Particularly with urls (including replacements `insteadOf`) it might not be clear what's the thing we resolved it to.

Reply:
I changed it, but then realized -- path is provided within `_msg`, so no need to duplicate, thus "It" is to stay ;)

Reply:
Ah, true. Sorry.